script.js

```diff
@@ -7,21 +7,23 @@ const allEpisodes = getAllEpisodes();
 
 function makePageForEpisodes(episodeList) {
   const display = document.getElementById("display");
-  display.innerText = `Displaying ${episodeList.length}/${allEpisodes.length}`;
+  display.innerText = `Display ${episodeList.length}/${allEpisodes.length}`;
```

Comment:
Please try to add display inner text outside the search bar, it looks much nicer.